src/lookupDataProvider.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import lookupTablesService from "./services/lookupTablesService";
+
+const LookupDataContext = React.createContext();
+class LookupDataProvider extends Component {
+    state = {
+        lookupData: null
+    }
+
+    render() {
+        return (
+            <LookupDataContext.Provider value={{
+                lookupData: this.state.lookupData,
+                loadLookupData: async () => {
```

Comment:
Presunut do lifecycle metody `componentDidMount()` - to je na to to spravne miesto :)